core/include/drivers/rtc.h

```diff
@@ -57,6 +57,18 @@ struct rtc_ops {
 	TEE_Result (*set_offset)(struct rtc *rtc, long offset);
 };
 
+#define RTC_TIME(year, mon, mday, wday, hour, min, sec, ms)		\
```

Comment:
We prefer have the macros being defined before the structs. Could you move these line before line 16?